marimo/_plugins/ui/_impl/table.py

```diff
@@ -220,6 +222,7 @@ def __init__(
         pagination: Optional[bool] = None,
         selection: Optional[Literal["single", "multi"]] = "multi",
         page_size: int = 10,
+        max_columns: Optional[int] = 50,
```

Comment:
This should be inserted after the `*` otherwise it'd a breaking change.

Reply:
Ah yes technically is. I'll fix that in a follow up. For 1.0, we should put most args behind the *